ui/pages/Staff/constants.js

```diff
@@ -90,6 +93,24 @@ const formatFamilySummary = row =>
     }
   </div>
 
+const formatIDLink = row =>
+  <div>
+    <Link to={`/project/${row.project_guid}/family_page/${row.family_guid}`} target="_blank">{row.family_id}</Link>
+  </div>
+
+const formatSuccessStoryTypes = row =>
+  row.success_story_types &&
+  <div>
+    {row.success_story_types.map(tag =>
+      <div>
+        <ColoredIcon name="stop" color={FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP[tag].color} />
+        {FAMILY_SUCCESS_STORY_TYPE_OPTIONS_LOOKUP[tag].name}
```

Comment:
You should have one shared component for displaying success story types, and that should be used everywhere they are displayed